soc/xtensa/intel_adsp/ace/power.c

```diff
@@ -315,6 +316,8 @@ __weak void pm_state_exit_post_ops(enum pm_state state, uint8_t substate_id)
 		soc_cpus_active[cpu] = true;
 		z_xtensa_cache_flush_inv_all();
 		z_xt_ints_on(core_desc[cpu].intenable);
+
+		sys_clock_idle_exit();
```

Comment:
I'm not sure but I think this should only be done for primary core and definitely only when IMR context saving is enabled.

Reply:
What you are resolving issues without fixing them?

Reply:
Sorry forgot to push.